### ISSUE TITLE
How to reproduce the result of few-shot CoOp on MS-COCO (Table 2 and Figure 6)

### ISSUE THREAD

**Issue by lee-plus-plus** [guozix/TaI-DPT#12]
I tried to train CoOp on multi-label **few-shot** settings with BCE loss and SGD optimizer with different learning rate, but failed. The CoOp only performs **worse** after few-shot prompt tuning. Is there any trick that I missed? 

By the way, I intialized the context of CoOp's prompt learner by "a photo of a", and no softmax is applied during the inference. 

Any suggestions would be greatly helpful.
